src/components/CommentList/CommentList.scss

```diff
@@ -1 +1,7 @@
-// add styles here
+.CommentList {
+  display: flex;
+  flex-direction: column;
+  gap: 0.7em;
+  background-color: #eee;
```

Comment:
You can move comment bg color into a scss variable. It improves readability and will be a good practice

Also, you can move other changeable properties to variables to have an easier way of changing it